lib/ash_authentication/verifier.ex

```diff
@@ -109,11 +109,23 @@ defmodule AshAuthentication.Verifier do
     end
   end
 
+  @spec is_token_resource(any()) :: boolean() | {boolean(), any()}
+  defp is_token_resource(module) when is_atom(module) do
+    with true <- function_exported?(module, :spark_is, 0),
```

Comment:
I would suggest doing `Spark.Dsl.is?(Ash.Resource)` :)

Reply:
That's exactly what I was looking for! Haha thanks